ponder-sh/ponder
# Adds Coverage

- Upgrade Vitest and related packages to latest versions
- Add @vitest/coverage-v8 for test coverage
- Update Docker Compose configuration for Postgres
- Add graphql resolution to ensure consistent version
- Update .gitignore to exclude coverage and postgres data
